bucket/qtranslate.json

```
@@ -0,0 +1,33 @@
+{
+    "homepage": "https://quest-app.appspot.com",
+    "description": "Translator utility",
+    "version": "6.7.3",
+    "license": {
+        "identifier": "Freeware",
+        "url": "https://quest-app.appspot.com/license"
+    },
+    "notes": "If you encounter 'Program initialization error', follow the instructions on 'https://quest-app.appspot.com/faq' to fix it.",
```

Small wording tweak. There is no need to reference back to problem in current context with `it`

```suggestion
    "notes": "If you encounter 'Program initialization error', follow the instructions on 'https://quest-app.appspot.com/faq'.",
```